x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -269,25 +233,6 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 		details["badge"] = fmt.Sprint(badgeSig)
 		details["clientFee"] = "0"
 		details["reliabilityPay"] = "false"
-		details["Mint"] = details["BasePay"]
```

Comment:
dont remove the event "Mint" it might break stuff for growth

Reply:
done. see https://github.com/lavanet/lava/commit/36c4bd192408759547b2588605992db3d3b23f21